micrometer-tracing/src/main/java/io/micrometer/tracing/Span.java

```diff
@@ -151,6 +166,30 @@ public Span remoteIpAndPort(String ip, int port) {
      */
     Span tag(String key, String value);
 
+    /**
+     * Sets a tag on this span.
+     * @param key tag key
+     * @param value tag value
+     * @return this span
+     */
+    Span tag(String key, long value);
```

Comment:
I think we should add `default` here and provide a default that converts to String cause otherwise we're making a breaking change